docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,130 @@
+
+NuGet packages target one of more frameworks. The .NET Standard Library packages target the ".NET Standard" framework, which is represented by the the `netstandard` Target Framework Moniker (TFM). The framework is ".NET Standard" instead of ".NET Standard Library", since ".NET Standard" includes both the .NET Standard Library packages and the larger ecosystem of libraries that are built on top of it. Libraries that are intended to run on multiple runtimes should use this target. 
+
+The `NETStandard.Library` metapackage references the complete set of NuGet packages that define the .NET Standard Library.  The most common way to target `netstandard` is by referencing this metapackage. It describes and provides access to the ~40 .NET  libraries and associated APIs that define the .NET Standard Library. You can reference additional packages that target `netstandard` to get access to additional APIs. 
+
+The `NETStandard.Library` metapackage doesn't define a framework of its own, but exposes the set of frameworks defined by the packages it references, including `netstandard`. Similarly, the `netstandard` framework does not expose any APIs on its own, but relies on packages that expose `netstandard` assets to define the framework. The definition of the `netstandard` framework is not limited to the .NET Standard Library.
+
+Note: `netstandard` isn't actually a framework, but a prototype for one. It is used as a short-hand. For example, `netstandard1.2` is an actual framework.
+
+Versioning
+----------
+
+The spec is not singular, but an incrementally growing and linearly versioned set of APIs. The first version of the standard establishes a baseline set of APIs. Subsequent versions add APIs and inherit APIs defined by previous versions. There is no established provision for removing APIs from the standard.
+
+The .NET Standard Library is not specific to any one .NET runtime, nor does it match the versioning scheme of any of those runtimes.
+
+APIs added to any of the runtimes (such as, .NET Framework, .NET Core and Mono) can be considered as candidates to add to the specification. New [versions of the .NET Standard Library](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#list-of-net-corefx-apis-and-their-associated-net-platform-standard-version) are created for one of two reasons:
+
+- APIs are added to a .NET runtime (for example, .NET Core) that are considered fundamental. They are added to the .NET Standard Library, requiring other .NET runtimes to implement these new APIs in order to satisfy the new standard version.
+- APIs are implemented across a set of .NET runtimes (for example, .NET Framework, .NET Core and Mono) at the same time, making the APIs candidates for the standard.
```

Comment:
Not sure the second bullet points adds much. The second bullet point describes a symptom that is indicative of the APIs being fundamental.